js/main.js

```diff
@@ -1,382 +0,0 @@
-// Weather App
-// by Roger Mullins
-
-// Object Definitions
-
-let stateObj = {
-  city : "",
-  zipCode: "",
-  tempK: 0,
-  tempF: 0,
-  tempC: 0,
-  condition: "",
-  imageURL: ""
-};
-
-let lookupHistoryObj = {
-  city: "",
-  zipCode: ""
-};
-
-let localStorageObj = {
-  city: "",
-  zipCode: ""
-}
-
-// Global Variable
-
-let zipIn = "";   // Used to validate ZIP
-let tempF = 0;    // Used to update temperature background color
-
-function buildUI() {
-  // Creates HTML elements
-
-  function createElement(parentID, typeOfElement, bootstrapClasses, idString, innerText = "") {
```

Comment:
In the future we could pass this as a single object to allow the parameters to be passed in any order. 

```javascript
function createElement({
  parentID = 'main',
  typeOfElement = 'div',
  bootstrapClasses = [],
  idString = '',
  innerText = "",
}) {
  // code goes here
}

// calling it
createElement({ bootstrapClasses: ['one', 'two'], idString: 'newDiv'})
createElement({ idString: 'twoDiv', innerText: 'two' })